.github/workflows/check-dist.yml

```diff
@@ -0,0 +1,53 @@
+# `dist/index.js` is a special file in Actions.
+# When you reference an action with `uses:` in a workflow,
+# `index.js` is the code that will run.
+# For our project, we generate this file through a build process from other source files.
+# We need to make sure the checked-in `index.js` actually matches what we expect it to be.
+name: Check dist/
```

Comment:
@t-botz The ending `/` is not necessary, github adds it by itself :)

Reply:
Ha ha, This is a copy paste from github.
https://github.com/actions/typescript-action/blob/main/.github/workflows/check-dist.yml#L6

I can remove